lib/src/sql/value/value.rs

```diff
@@ -137,6 +137,7 @@ impl Default for Value {
 }
 
 impl From<bool> for Value {
+	#[inline]
```

Comment:
This is now, more than ever, a hot code path. `#[inline]` should guard against function-call overhead.

Reply:
On a side note, for future, there are probably many more functions which could benefit from `#[inline]`ing!